sale_layout_category_hide_detail/views/account_move_view.xml

```diff
@@ -6,8 +6,9 @@
         <field name="name">account.move.form.inherit</field>
         <field name="model">account.move</field>
         <field name="inherit_id" ref="account.view_move_form" />
+        <field name="priority" eval="75" />
```

Comment:
Why is this priority added in this view? It shouldn't be necessary.

Reply:
In deed, it is not necessary